theano/tensor/signal/pool.py

```diff
@@ -799,11 +1034,11 @@ def __init__(self, ignore_border, mode='max', ndim=2, openmp=None):
     def prepare_node(self, node, storage_map, compute_map, impl):
         if len(node.inputs) < 5:  # 5 for AveragePoolGrad, 6 for MaxPoolGrad
             # Old interface
-            self.ndim = len(node.op.ds)
+            self.ndim = len(node.op.ws)
             self.mode = node.op.mode
-            ws = theano.tensor.constant(node.op.ds)
-            st = theano.tensor.constant(node.op.st)
-            pad = theano.tensor.constant(node.op.padding)
```

Comment:
We should revert that, because the old serialized Ops will still have their attributes named 'ds', 'st' and 'padding'